src/main/java/mate.academy.internetshop/Main.java

```diff
@@ -0,0 +1,92 @@
+package mate.academy.internetshop;
+
+import java.util.List;
+import mate.academy.internetshop.lib.Inject;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Bucket;
+import mate.academy.internetshop.model.Item;
+import mate.academy.internetshop.model.Order;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
+
+public class Main {
```

Comment:
do we need main class in this repository?

Reply:
Removed this class from the project